odata-to-abstract-sql.ometajs

```diff
@@ -343,7 +343,7 @@ export ometa OData2AbstractSQL {
 			.map(odataNameToSqlName)
 			.flatMap(function(sqlName) {
 				return $elf.Synonym(sqlName).split('-')
-			}).join('.')
+			}).value()
```

Comment:
`_.get` accepts either `a.b.c` or `[ 'a', 'b', 'c' ]` and the latter is much faster as it avoids a load of parsing